### ISSUE TITLE
Android development environment setup

### ISSUE THREAD

**Issue by paulmelis** [StereoKit/StereoKit#505]
## Describe the feature

I'm not familiar with Android development, but I suspect one the NDK and such are set up StereoKit app development will not be really that different than for, say, HoloLens or PC. However, I'm having trouble setting all the dev stuff up correctly. I did find this page: https://petesprojects.net/How-to-Stereokit-and-Android/. This got me a little further, although it's 1.5 years old and I wonder if everything there is still up-to-date. Also, the single comment adds some detail on where to run some of the commands (which was indeed something I got stuck on).

But even after all this I still get a build error for `StereoKitTest_Android` (will add later, get it on my win laptop, not linux machine I'm editing this on). So all in all it would be fantastic to have a one-pager on setting up a Windows system for Android development with SK in the docs.

## An example use-case

An easy start of SK development for Android devices, e.g. Quest 2, Quest Pro

**Comment by maluoi** [comment]
Yeah, this would be really good to have. The main reason I haven't yet is because I'm not much of an Android dev myself, I'm not 100% sure how I actually got it all working, and it may change in the near future!

I _am_ actively working on improving the dev experience for Android, both on the SDK dev side, and the product dev side. In particular I'd like to get the build process working nicely on GitHub Actions, which will definitely require nailing down build steps explicitly :D So probably expect some related docs in the near future!
